packages/common/src/add-name-to-message.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ Adds first and last name when First Name and Last Name fields lose focus if name shortcodes aren't present
+*/
+
+import { ENGrid } from ".";
+
+export class addNameToMessage {
```

Comment:
For class names, we use PascalCase. Please change it to AddNameToMessage.

Reply:
I've changed the class name.